packages/core/strapi/lib/services/__tests__/entity-validator.test.js

```diff
@@ -592,4 +589,685 @@ describe('Entity validator', () => {
       });
     });
   });
+
```

Comment:
Nice suite of tests! The file becomes pretty big, what do you think of splitting it in different test files?